test/test_fiat_p256.ml

```diff
@@ -0,0 +1,41 @@
+module Testable = struct
+  let cstruct =
+    let pp fmt t = Hex.pp fmt (Hex.of_cstruct t) in
+    Alcotest.testable pp Cstruct.equal
+end
+
+let list_init l f =
+  let rec go acc i =
+    if i = l then
+      List.rev acc
+    else
+      go ((f i)::acc) (i + 1)
+  in
+  go [] 0
+
+let int32_to_hex i = Printf.sprintf "%08Lx" i
+
+let prng len =
+  let i32_count = len / 4 + 1 in
+  let i32s = list_init i32_count (fun _ -> Random.int64 0x100000000L) in
```

Comment:
I think you'll want a toplevel `let () = Random.self_init ()` somewhere (that does not matter too much).